plugins/source/azure/client/resolvers.go

```diff
@@ -4,9 +4,14 @@ import (
 	"context"
 
 	"github.com/cloudquery/plugin-sdk/schema"
+	"github.com/thoas/go-funk"
 )
 
 func SubscriptionIDResolver(_ context.Context, meta schema.ClientMeta, r *schema.Resource, col schema.Column) error {
 	cl := meta.(*Client)
 	return r.Set(col.Name, cl.SubscriptionID)
 }
+
+func ParentIDResolver(_ context.Context, _ schema.ClientMeta, r *schema.Resource, col schema.Column) error {
+	return r.Set(col.Name, funk.Get(r.Parent.Item, "ID"))
```

Comment:
I think we could use the SDK's `schema.ParentColumnResolver` utility here WDYT?

Reply:
updated